embedded-redis/src/test/resources/bootstrap-noauth.properties

```diff
@@ -0,0 +1 @@
+embedded.redis.requirepass=false
```

Comment:
I'd better specify this config in the test's properties. You could get rid of profiles and profile-related property files this way. You can also remove intermediate abstract classes: BaseClusterEmbeddedRedisTest  and BaseEmbeddedRedisTest - just extract config in BaseClusterEmbeddedRedisTest into separate file.
The implementation is more complex now due to unnecessary reuse.

Reply:
i'll leave structure by now as it is, as discussed